### ISSUE TITLE
[web] Hive 4.0.0-dev.2 - issues initializing

### ISSUE THREAD

**Issue by bhagn** [hivedb/hive#1239]
Hello - I know that 4.0.0 is still a pre-release. I encountered a few issues while getting started on web.

The first issue is it seems to be expecting a directory:
```
Invalid argument(s) (directory): No directory specified and no defaultDirectory provided.
```

If we set the `Hive.defaultDirectory = ''` it seems to move ahead, but this time we get a different error:

```
On web you have to call Isar.initialize() manually before using Isar.
```

Calling `Isar.initialize()` throws the following error:
```
IsarError: Operation not supported. You might need to use a different flavor of Isar.
```

Does this mean we need to install `Isar` as well? Is there documentation that can help setup Hive for web?

**Comment by karvulf** [comment]
Same issue here

**Comment by brasizza** [comment]
Same here =(

**Comment by p4-k4** [comment]
Same here

**Comment by p4-k4** [comment]
https://github.com/isar/hive/issues/1225#issuecomment-1691219875

**Comment by bhagn** [comment]
@p4-k4 I don't think the comment addresses the issue mentioned here. The comment only says that web won't support persistance - just in-memory, until a stable release.

**Comment by p4-k4** [comment]
> @p4-k4 I don't think the comment addresses the issue mentioned here. The comment only says that web won't support persistance - just in-memory, until a stable release.

Nah, never said it addresses the issue. But is the closest somewhat official response to this issue - whether or not it has anything to do with why this issue exists, I have no idea. Maybe web overall is only half-implemented for `4.0.0-dev`. Who knows.

I see there's another pending issue posted about this quite some time ago too.

Hang tight, I'd say.
